headless.txt

```diff
@@ -0,0 +1,6 @@
+# The contents of this file are automatically appended to Dockerfile
+
+RUN apt-get update \
+  && apt-get install -y chromium --no-install-recommends \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
I wonder if we should add `/var/cache/apt/*` to this `rm -rf`? Or maybe `apt-get clean` does that for us?

Reply:
[Docs](https://linux.die.net/man/8/apt-get) say:

> Clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/.

Sounds to me like that is enough 🤷‍♂️